gcp_variant_transforms/testing/integration/run_preprocessor_tests.py

```diff
@@ -76,6 +77,14 @@ def __init__(self,
             '--temp_location {}'.format(parser_args.temp_location),
             '--job_name {}'.format(
                 ''.join([test_name, suffix]).replace('_', '-'))]
+    self._header_blob_name = ''
```

Comment:
nit: prefer using None instead of empty string.

Reply:
Done.